operations/security_and_authentication/Add_LDAP_User_Federation.md

```diff
@@ -205,59 +248,216 @@ LDAP user federation is not currently configured in Keycloak. For example, if it
           - type: string
           - allowed values: true or false
         * ldapRoleMapperClientId: If ldapRoleMapperUseRealmRolesMapping is false then this is the client ID to apply the roles to.
-          - default: shasta
-          - type: string
+            - default: shasta
+            - type: string
+      ```
+
+3. Prepare to generate Sealed Secrets.
+   
+   Secrets are stored in customizations.yaml as `SealedSecret` resources 
+   (encrypted secrets), which are deployed by specific charts and decrypted by the
+   Sealed Secrets operator. But first, those Secrets must be seeded, generated, and
+   encrypted.
+
+   1. Mount the PITDATA so that helm charts are available for the re-install (it might already be mounted).
+
+      ```bash
+      ncn-m001# mkdir -pv /mnt/pitdata
+      ncn-m001# mount -L PITDATA /mnt/pitdata
+      ```
+   
+   2. Load the `zeromq` container image required by Sealed Secret Generators.
+
+      > **NOTE:** A properly configured Docker or Podman environment is required.
+
+      ```bash
+      ncn-m001# /mnt/pitdata/${CSM_RELEASE}/hack/load-container-image.sh dtr.dev.cray.com/zeromq/zeromq:v4.0.5
+      ```
+
+   3. Re-encrypt the existing secrets:
+
+      ```bash
+      ncn-m001# /mnt/pitdata/prep/site-init/utils/secrets-reencrypt.sh customizations.yaml \
```

Comment:
What directory are we in at this point?   Should they be using the sealed_secrets.key and sealed_secrets.crt from /root/site-init/certs?   

If we are expecting them to be in `/root/site-init` (where customizations.yaml is located), then I think this command could just be `./utils/secrets-reencrypt.sh customizations.yaml ./certs/sealed_secrets.key ./certs/sealed_secrets.crt`